pkg/mic/mic.go

```diff
@@ -151,6 +160,8 @@ func (c *Client) Sync(exit <-chan struct{}) {
 		}
 		stats.Put(stats.System, time.Since(systemTime))
 
+		nodeRefs := make(map[string]bool)
```

Comment:
can we please rename nodeRefs to something which represents the usage requirement-  as in we are trying to grab the nodes which potentially have identities which are still in use because a binding exists, so could we please  name this to reflect this usage.

Reply:
Do you have a suggestion?
`nodeRefs` fits the bill for me, especially since this is all nodes with a pod that has an identity.